### ISSUE TITLE
Custom board views for tasks in a project

### ISSUE THREAD

**Issue by ahamdi** [Meeds-io/meeds#1395]
**Is your feature request related to a problem? Please describe.**
Sometimes when managing a project, we need to have different views for the same bunch of tasks that will give us better idea on the evolution of a specific project version in the same project. Currently we are obliged to duplicate tasks to manage a specific version  or a specific feature in a project whereas it will be possible to do that using filtering on some labels to create the needed custom view.

**Describe the solution you'd like**
Using the tasks filter, we can create a persisted views that we can use to follow a specific version or feature in working projects.

**Describe alternatives you've considered**

**Additional context**

**Comment by ahamdi** [comment]
@srenault-meeds is this a refused status ?

**Comment by srenault-meeds** [comment]
I did nothing. The task is automatically archived from the moment there are no updates during 1 month I think. This is it 🙂

**Comment by ahamdi** [comment]
Ok, I see. then is there an update for this suggested improvement ? is it something interesting ?

**Comment by srenault-meeds** [comment]
Yes, looks interesting indeed. It reminds me something we can do here in github. Now, I have no ownership to tasks. As you might know, this is shared between eXo and MeedsLab and overall Meeds DAO.

Anyway, yes it looks interesting and could be useful to propose to users to create view
